torch/ao/quantization/_pt2e/quantizer/x86_inductor_quantizer.py

```diff
@@ -154,10 +175,202 @@ def _annotate_for_static_quantization_config(
         # and we will mark the matched node with "_annoated" so fusion operator pattern
         # can take precedence over single operator pattern in this way
         config = self.global_config
+        self._annotate_conv2d_binary_unary(model, config)
+        self._annotate_conv2d_binary(model, config)
+        self._annotate_conv2d_unary(model, config)
         self._annotate_conv2d(model, config)
 
         return model
 
+    def _annotate_conv2d_binary_unary(
```

Comment:
Possible to factor out common code from these annotation functions?

Reply:
Thanks for the suggestions. Put out common code into the help functions. And per the discussion here: https://github.com/pytorch/pytorch/pull/98730#discussion_r1229891161, we can follow up to use the common utils provided by quantization when these utils are available.